src/server/cards/adventures/CoinOfTheRealm.js

```diff
@@ -0,0 +1,19 @@
+import Card from 'cards/Card';
+
+export default class CoinOfTheRealm extends Card {
+  static cost = new Card.Cost({ coin: 2 });
+  static types = new Set(['Treasure', 'Reserve']);
+  onPlay(player) {
+    player.money++;
+    player.moveCard(this, player.playArea, player.mats.tavern);
```

Comment:
Might want to consider making functions that move cards to/from the tavern mat. Would allow us to do stuff like:
```js
calledReserve(card, to = this.playArea) {
    this.moveCard(this, player.mats.tavern, to);
    this.game.log(`${this.name} calls {$card.title} from their tavern mat);
}

I don't know if there is anything else we might want to put in there.